### PR TITLE
Hide exercises in progress

### PR DESCRIPTION
**Pull request by tomlovesgithub**

### CODE REVIEW

**Review by lloback — CHANGES_REQUESTED** [review]
@tomlovesgithub  can you explain to me what the code is doing? What is the ticket number? I don't see any major change in the code.

**Comment by tomlovesgithub** [comment]
> @tomlovesgithub can you explain to me what the code is doing? What is the ticket number? I don't see any major change in the code.

This is the request from Jeremy to comment out the 'Exercises in progress' section from the vacancy page https://judicial-appointments.slack.com/archives/CLV1P6D6J/p1601924140006800

**Comment by lloback** [comment]
Ok, I can see now what it is doing. It is not a good practice to merge code into the repo without a ticket number. You will be here for a while and it is important that you can go back and check the reason for the change. I had to ask you, because it was not clear to me.
Did Jeremy created the ticket? If not, you can create yourself and add that information here.

File: src/views/Vacancies.vue [hunk 1]
```diff
@@ -204,7 +204,7 @@
           </ul>
         </div>
 
-        <div class="inProgressApplicationsList">
+        <!-- <div class="inProgressApplicationsList">
```
Comment (lloback) [inline src/views/Vacancies.vue:207]: There is another way of doing this on VUE which I think it is cleaner:

`v-if="false"

It hides it completely from the DOM

**Review by lloback — APPROVED** [review]

**Pull request merged**